### ISSUE TITLE
[FR] Rethink the use of `usethis::create_project()`

### ISSUE THREAD

**Issue by ColinFay** [ThinkR-open/golem#1149]
I'm unsure if we really need to call `usethis::create_project` at project creation.

**Comment by pachadotdev** [comment]
@ColinFay should this be closed? I see the only file with `create_project` is NEWS.md in 0.6.0

**Comment by ColinFay** [comment]
@pachadotdev you're right! This has been implemented in https://github.com/ThinkR-open/golem//commit/7d748b81ba977bac1c7aba0ffd5f1f7b06b3b6f6
